judge/views/organization.py

```diff
@@ -85,18 +85,22 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class OrganizationUsers(OrganizationDetailView):
+class OrganizationUsers(QueryStringSortMixin, OrganizationDetailView):
     template_name = 'organization/users.html'
+    all_sorts = frozenset(('points', 'problem_count', 'rating', 'performance_points'))
```

Comment:
Is it even possible to sort by points from the UI?

Reply:
I wasn't sure what it does and it's present on the main user list, so I felt leaving it in would be wise.